pages/arcade.liquid

```diff
@@ -0,0 +1,1304 @@
+            <p class="title font-kufam text-pink">{{ rebel.position }}</p>
+            <p class="testimonial font-kufam text-white">"{{ rebel.testimonial }}""</p>
+          </div>
+        </div>
+      {% endfor %}
+    </div>
+    {% for quest in quests %}
+      {% assign parent_index = forloop.index %}
+      <section id="level-{{parent_index}}-quests" class="level-quest font-kufam">
+        <div class="d-flex justify-content-center align-items-center">
+          {% for i in (1..parent_index) %}
+            <img
+              class="level-{{parent_index}}"
+              src="/img/arcade/treasure-chest.webp"
+              alt="treasure chest"
+            >
+          {% endfor %}
+        </div>
+        <p class="overline text-pink font-kufam text-uppercase text-center">{{ quest.name }}</p>
+        <h2 class="text-white font-horsemen text-center mx-auto">{{ quest.heading }}</h2>
+        <div class="level-quest-container text-white">
+          <div class="level-heading d-none d-md-flex justify-content-start align-items-center">
+            <div>
+              {% for i in (1..parent_index) %}
+                <img src="/img/svg/pixel-heart.svg" alt="heart">
+              {% endfor %}
+            </div>
+            <h3 class="font-horsemen text-uppercase">Level {{ forloop.index }}</h3>
+          </div>
+          <p>{{ quest.description }}</p>
+          <img class="d-none d-sm-block" src="{{quest.image}}" alt="{{quest.name}}">
+          <img class="d-block d-sm-none w-100" src="{{quest.image-mobile}}" alt="{{quest.name}}">
+
+          <div class="products-container">
+            {% for product in quest.products %}
+              <div class="product border-pink d-flex flex-column font-kufam">
+                <div class="level-heading d-flex align-items-center">
+                  <div>
+                    {% for i in (1..parent_index) %}
+                      <img src="/img/svg/pixel-heart.svg" alt="Heart">
+                    {% endfor %}
+                  </div>
+                  <h4 class="font-horsemen text-uppercase mb-0">Level {{ parent_index }}</h4>
+                </div>
+                <img class="d-none d-sm-block" src="{{product.image}}" alt="{{product.name}}">
+                <img
+                  class="d-block d-sm-none w-100"
+                  src="{{product.image-mobile}}"
+                  alt="{{product.name}}"
+                >
+                <h5 class="name text-green">{{ product.name }}</h5>
+                <p class="price text-pink text-italic">${{ product.price }}</p>
+                <p class="text-white">{{ product.description }}</p>
+                <a
+                  href="{{product.link}}"
+                  class="btn btn-pink font-kufam mt-auto d-block"
+                  target="_blank"
+                  {% if product.link == '' %}
+                    disabled
+                  {% endif %}
+                  >Buy Now</a
```

Comment:
We also chatted a bit about this verbiage. Instead of `Buy Now` maybe something like `Unlock` or something similar